mlflow/tracking/fluent.py

```diff
@@ -357,10 +573,35 @@ def create_experiment(name, artifact_location=None):
     """
     Create an experiment.
 
-    :param name: The experiment name. Must be unique.
+    :param name: The experiment name, which must be unique and is case sensitive
     :param artifact_location: The location to store run artifacts.
                               If not provided, the server picks an appropriate default.
-    :return: Integer ID of the created experiment.
+    :return: String ID of the created experiment.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create an experiment name, which must be unique and case sensitve
+        experiment_id = mlflow.create_experiment("Social NLP Experiments")
+        experiment = mlflow.get_experiment(experiment_id)
+
+        # Print the contents of experiment data
+        print("Name: {}".format(experiment.name))
+        print("Experiment_id: {}".format(experiment.experiment_id))
+        print("Artifact Location: {}".format(experiment.artifact_location))
+        print("Tags: {}".format(experiment.tags))
+        print("Lifecycle_stage: {}".format(experiment.lifecycle_stage))
+
+    .. code-block:: text
+        :caption: Output
+
+        Name: Social NLP Experiments
+        Experiment_id: 1
+        Artifact Location: file:///.../apis/mlruns/1
```

Comment:
```suggestion
        Artifact Location: file:///.../mlruns/1
```

Reply:
Yes, better to show only `file:///.../mlruns/1`